### DISCUSSION TITLE
New App Garden!! 🪴

### DISCUSSION THREAD

**Discussion by Jeffser** [Jeffser/Alpaca#308] [Announcements]
Hiii, today I'm releasing the first early alpha build of my new app [Garden](https://github.com/Jeffser/Garden)!

Garden is a Shimeji manager, they are little characters that run around on your desktop, think of them as pc pets

![image](https://github.com/user-attachments/assets/061bca29-4740-4699-90eb-71a6e37a4d40)

![image](https://github.com/user-attachments/assets/edd3c475-536f-4e12-8806-2e5671e0907b)

**Comment by olumolu** [comment]
What it does?
